.golangci.yaml

```diff
@@ -0,0 +1,19 @@
+# https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml
+run:
+  timeout: 10m
+
+linters-settings:
+  govet:
+    settings:
+      printf:
+        funcs:
+          - Infof
+          - Debugf
+          - Warnf
+          - Errorf
+          - Fatalf
+          - Panicf
+          - DPanicf
+    disable-all: true
```

Comment:
Why not include them if we are passing all of the analyzers? (Tested locally and appears to pass)

Reply:
Missed this one